tests/test_stubs.py

```diff
@@ -50,14 +51,14 @@ def test_fail(path, py2_arg):
     with open(path) as fin:
         lines = fin.readlines()
 
-    errors = {}
+    errors = defaultdict(lambda: "")
```

Comment:
The previous iteration of the code didn't work too well when we had multi-line error messages (e.g. with overload matching errors), as it would just overwrite the error messages.